src/redisearch_rs/trie_bencher/benches/operations.rs

```diff
@@ -1,26 +1,72 @@
 //! Benchmark the core operations provided by a trie map—insertions, deletions, and lookups.
 //!
 //! Refer to `data/bench_trie.txt` to visualize the structure of the trie that's being benchmarked.
-use std::path::PathBuf;
+
+use std::time::Duration;
 
 use criterion::{Criterion, criterion_group, criterion_main};
 use trie_bencher::OperationBencher;
 
-fn criterion_benchmark(c: &mut Criterion) {
-    let file_path = PathBuf::from("data").join("bench.txt");
-    let contents = fs_err::read_to_string(file_path).unwrap();
-
-    let mut bencher = OperationBencher::new(c, contents.clone());
-    bencher.load_group();
-    bencher.insert_group("colder", "Insert (leaf)");
-    bencher.insert_group("fan", "Insert (split with 2 children)");
-    bencher.insert_group("effo", "Insert (split with no children)");
-    bencher.find_group("form", "Find no match");
-    bencher.find_group("April,", "Find match (depth 2)");
-    bencher.find_group("enormous", "Find match (depth 4)");
-    bencher.remove_group("bright", "Remove leaf (with merge)");
-    bencher.remove_group("along", "Remove leaf (no merge)");
+use trie_bencher::corpus::CorpusType;
+
+fn criterion_benchmark_gutenberg(c: &mut Criterion) {
+    let corpus = CorpusType::GutenbergEbook;
+    let keys = corpus.create_keys(true);
+
+    let bencher = OperationBencher::new("Gutenberg".to_owned(),  keys, None);
+    bencher.load_group(c);
+    bencher.insert_group(c, "colder", "Insert (leaf)");
+    bencher.insert_group(c, "fan", "Insert (split with 2 children)");
+    bencher.insert_group(c, "effo", "Insert (split with no children)");
+    bencher.find_group(c, "form", "Find no match");
+    bencher.find_group(c, "April,", "Find match (depth 2)");
+    bencher.find_group(c, "enormous", "Find match (depth 4)");
+    bencher.remove_group(c, "bright", "Remove leaf (with merge)");
+    bencher.remove_group(c, "along", "Remove leaf (no merge)");
+}
+
+fn criterion_benchmark_redis_wiki_1k(c: &mut Criterion) {
+    let corpus = CorpusType::RedisBench1kWiki;
+    let keys = corpus.create_keys(true);
+
+    let bencher = OperationBencher::new("Wiki-1K".to_owned(), keys, None);
+    bencher.load_group(c);
+
+    // parent at line 95 in redis_wiki1k_titles_bench.txt
+    bencher.insert_group(c, "Abigail", "Insert (Split with 2 children)");
+    // parent at line 1 (root) in redis_wiki1k_titles_bench.txt
+    bencher.insert_group(c, "Zoo", "Insert (Split with 18 children)");
```

Comment:
```suggestion
    bencher.insert_group(c, "Abigail", "Insert (split with 2 children)");
    // parent at line 1 (root) in redis_wiki1k_titles_bench.txt
    bencher.insert_group(c, "Zoo", "Insert (split with 18 children)");
```

Let's use consistent capitalization across bench names.